xc/common/primitives/ologice3/ologice3.pb_type.xml

```diff
@@ -206,23 +371,14 @@
       <direct name="TBYTEOUT" input="OSERDESE2.TBYTEOUT" output="OLOGICE3.TBYTEOUT" />
       <direct name="TCE" input="OLOGICE3.TCE" output="OSERDESE2.TCE" />
       <direct name="TFB" input="OSERDESE2.TFB" output="OLOGICE3.TFB" />
-      <direct name="TQ" input="OSERDESE2.TQ" output="OLOGICE3.TQ" />
       <direct name="SHIFTIN1" input="OLOGICE3.SHIFTIN1" output="OSERDESE2.SHIFTIN1" />
       <direct name="SHIFTIN2" input="OLOGICE3.SHIFTIN2" output="OSERDESE2.SHIFTIN2" />
+
+      <direct name="T1_to_T_INV.TI" input="OLOGICE3.T1" output="T_INV.TI"/>
+      <mux name="TQ" input="OSERDESE2.TQ T_INV.TO" output="OLOGICE3.TQ"/>
```

Comment:
Don't you need a "fasm_mux" annotation here?

Reply:
No. When the route though the `T_INV` is chosen this inverter is active and that corresponds to not emitting the `ZINV_T1` feature. The feature disables the inversion. When the `OSERDES` is used then the inversion is controlled with its parameters.

Reply:
Can you add what you wrote as an XML comment please?  I don't think this case is obvious to the reader.

Reply:
Mhm, added the comment.

Reply:
Thanks